src/get-started/test-service.md

```diff
@@ -177,22 +186,22 @@ failures:
 
 > **Test:** `test_transfer`
 
-Let’s test a transfer between two wallets. First, we need to create the testkit
-and the wallets:
+Let’s test a transfer between two wallets. First, we need to create the
+testkit and the wallets:
```

Comment:
Let’s test a transfer between two wallets. To do this, first, we need to initialize the testkit. Then we need to create the wallets and transfer funds between them. As per the code of the Cryptocurrency Service, the wallets are created with the initial balance set to `100`. 

The mentioned three transactions will be included into the block: